backend/src/controllers/utils/res-helpers.ts

```diff
@@ -0,0 +1,24 @@
+import { Request, Response } from "express";
+
+export interface Requester {
+  (): void;
+}
+
+export function setSuccess(res: Response, data?: any): void {
```

Comment:
Since we don't want any in our project, we all have a type :)
Here's how we can improve it:
```suggestion
export function setSuccess<T>(res: Response, data?: T): void {
```

And when you use it you can do:
```
setSuccess<yourType>(...)
```

Reply:
boooooooo

Reply:
vednash..